jobs/ci-fejta-soak-test.sh

```diff
@@ -19,9 +19,9 @@ set -o pipefail
 set -o xtrace
 
 # Hack to test use a test image
-OVERRIDE="${WORKSPACE}/hack/jenkins/.kubekins_e2e_image_tag"
-mkdir -p $(dirname "${OVERRIDE}")
-echo v20161116-fc014f3 > "${OVERRIDE}"
+#OVERRIDE="${WORKSPACE}/hack/jenkins/.kubekins_e2e_image_tag"
```

Comment:
Since I'm updating this image I no longer need to use a different version for these tests